### PR TITLE
Correctly display users ski/snowboard status on profile page

### PR DESCRIPTION
**Pull request by ganznz**

### CODE REVIEW

**Review by bcho892 — APPROVED** [review]
just a note, should include that it closes #406, in the "development" tab on the right

File: client/src/pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -23,6 +25,24 @@ const SignOutButton = () => {
   )
 }
 
+const determineUserSkiSnowboardStatus = (
+  currentUserData?: UserAdditionalInfo
```
Comment (bcho892) [inline client/src/pages/Profile/Profile.tsx:29]: also nit: prefer pass by value when possible (just including the skiier and snowboarding) so the function can't "accidentally" mutate the object

Comment (ganznz) [reply client/src/pages/Profile/Profile.tsx]: Updated, let me know if that's better before I merge

**Review by bcho892 — APPROVED** [review]
lgtm

**Pull request merged**